pkg/cmdutil/validate.go

```diff
@@ -0,0 +1,10 @@
+package cmdutil
+
+func ValidateEnumField(inputVal string, allowedValues []string, err error) error {
```

Comment:
🛷 Slick 🛷

Reply:
Interesting, it works for me if I do these steps:
1. remove the old config file by running `rm -r /path/to/.pdagent`
2. to test add this line: `s.logger.Infof("Server will be using this events url: %v", common.PdEventsUrl())` to the beginning of `func (s *Server) Start() error` in `pkg/server/server.go`
3. build the new pdagent binary with `make build`
4. run `pdagent init`
5. open the config file in VSCode: `code /path/to/.pdagent`
6. edit the region field to be `eu` instead of `us` and save the file
7. run `pdagent server`

When the server starts up I see this in the server logs: 
```
2021-07-12T13:12:51.817-0400    INFO    Server  server/server.go:62     Server will be using this events url: https://events.eu.pagerduty.com
```

Are you doing anything differently? Would love to test you method out to see what's going wrong.